superset-frontend/src/explore/components/controls/AnnotationLayerControl/AnnotationLayer.jsx

```diff
@@ -29,22 +29,19 @@ import {
   validateNonEmpty,
 } from '@superset-ui/core';
 
-import SelectControl from './SelectControl';
-import TextControl from './TextControl';
-import CheckboxControl from './CheckboxControl';
-
+import SelectControl from 'src/explore/components/controls/SelectControl';
+import TextControl from 'src/explore/components/controls/TextControl';
+import CheckboxControl from 'src/explore/components/controls/CheckboxControl';
```

Comment:
I personally prefer relative imports if they're in the same directory, but I don't have strong opinions on this. But it would be nice if we could enforce this with with some linting rules, e.g. requiring relative imports if in the same or parent directory, and absolute if elsewhere.

Reply:
@villebro I agree with you. I'm using relative imports if they're in the same directory and full paths otherwise. Since in this case, they are not in the same directory anymore, I used full paths. Totally agree with the Lint rules.

Reply:
My apologies, I didn't notice that this file moved 🤦 My "internal" linter is ok with relative imports from parent directories, but absolute is also just fine! But it would be great if the linter enforced fully deterministic imports (will try to poke around to see if this is possible/makes sense)

Reply:
No problem at all! 😄 

> But it would be great if the linter enforced fully deterministic imports (will try to poke around to see if this is possible/makes sense)

This would be awesome!

@villebro @rusackas I think that after [SIP-61](https://github.com/apache/superset/issues/13632) we'll have a more predictable structure, and that can allow us to use [Webpack module aliases](https://webpack.js.org/configuration/resolve/) for our main folders and help with full imports.